src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -42,6 +42,15 @@ navigator_params.add("default_avoid_dist", double_t, 0, "brief description", 50,
 navigator_params.add("collision_avoid_velocity_scale", double_t, 0, "brief description", .5, 0,  1)
 # add more navigator parameters here
 
+#######################################################################
+#                              PASSING                                #
+#######################################################################
+
+navigator_params = gen.add_group("Passing")
+navigator_params.add("static_position_quality_x_offset", double_t, 0, "", 0.3, 0, 1)
```

Comment:
These names should probably specify these are the _field_ x and y offsets, just to be extra explicit. The `Parameter` variable names should be updated accordingly.

Reply:
changed